core/src/main/java/io/lionweb/lioncore/java/model/impl/M3Node.java

```diff
@@ -72,7 +73,8 @@ public Object getPropertyValue(Property property) {
    * This internal method uses a property name and not a property or the property id because of a
    * circular dependency problem present for nodes representing M3 elements.
    */
-  protected <V> V getPropertyValue(String propertyName, Class<V> clazz, V defaultValue) {
+  protected <V> V getPropertyValue(
```

Comment:
mark return and `defaultValue` `@Nullable`?

Reply:
Done, thank you